apis/extension/scheduling.go

```diff
@@ -64,6 +64,14 @@ const (
 	GangModeNonStrict = "NonStrict"
 )
 
+const (
+	QuotaKoordinatorPrefix = "quota.scheduling.koordinator.sh"
+	LabelQuotaIsParent     = QuotaKoordinatorPrefix + "/is-parent"
+	LabelQuotaParent       = QuotaKoordinatorPrefix + "/parent"
+	LabelAllowLentResource = QuotaKoordinatorPrefix + "/allow-lent-resource"
```

Comment:
Share resource is the default behavior, therefore, I think it is easier to understand with the negative name, such as calling: disable-share-resource ？ @buptcozy